libherokubuildpack/README.md

```diff
@@ -0,0 +1,37 @@
+# libherokubuildpack &emsp; [![Docs]][docs.rs] [![Latest Version]][crates.io] [![MSRV]][install-rust]
+
+Common utilities for buildpacks written with [libcnb.rs](https://github.com/heroku/libcnb.rs). Originally designed to be
+only used for official Heroku buildpacks. It was moved into the libcnb.rs repository as an incubator for utilities that
+might find their way into libcnb.rs proper.
+
+This crate is optional and not required to write buildpacks with libcnb.rs. It provides helpers that buildpack authors 
+commonly need. Examples are digest generation, filesystem utilities, HTTP download helpers and tarball extraction. 
+
+## Crate Features
+
+It is common to not need all the helpers in this crate. To avoid including unnecessary code and dependencies, this crate
+uses Cargo features to allow opt-out of certain modules if they're not needed.
+
+The feature names line up with the modules in this crate. All features are enabled by default.
```

Comment:
This says all features are enabled by default. However even now, only some features are enabled by default. (And we may also want to disable all by default, pending the other comment.)

Reply:
Not all of them being enabled is an oversight on my part. :) Let's continue the all enabled as default discussion on the other comment. :)

Reply:
Enabled all of them as part of [bdebdbf](https://github.com/heroku/libcnb.rs/pull/495/commits/bdebdbfa79bf3f7fe479d498b1d78b119a9bd7fc)